app/assets/stylesheets/shared/tooltips.scss

```diff
@@ -0,0 +1,76 @@
+.tooltip-item {  
+  $base-border-color: gainsboro !default;
+  $base-border-radius: 3px !default;
+  $base-line-height: 1.5em !default;
+  $dark-gray: #333 !default;
+  $base-font-color: $dark-gray !default;  
+  $tooltip-background: white;
+  $tooltip-color: $base-font-color;
+  $tooltip-max-width: 16em;
+  $tooltip-arrow-width: 8px;
+  $tooltip-shadow: 0 2px 2px silver;
+  $tooltip-distance-from-item: 5em;
+  $tooltip-arrow-distance-from-box: -1.2em;
+
+  // background: $tooltip-background;
+  // border-radius: $base-border-radius;
+  // border: 1px solid $base-border-color;
+  // display: inline-block;
+  // padding: 0.5em 1em;
+  // position: relative;
+  // text-align: center;
+
+  &:focus,
+  &:hover .tooltip {
+    opacity: 1;
+    visibility: visible;
+  }
+
+  .tooltip {
+    @include position(absolute, null 0 $tooltip-distance-from-item 0);
+    @include transition (all 0.2s ease-in-out);
+    background-color: $tooltip-background;
+    background: #fff;
+    border-radius: $base-border-radius;
+    box-shadow: $tooltip-shadow;
+    color: $tooltip-color;
+    font-size: 0.9em;
+    // Make sure you use -webkit-backface-visibility: hidden; on the body element to prevent 1px nudging bugs.
+    line-height: 1.5em;
+    margin: 0 auto 0 -1em;
+    max-width: $tooltip-max-width;
+    width: 14em;
+    opacity: 0;
+    padding: 1em;
+    text-align: center;
+    visibility: hidden;
+    z-index: 10;
+
+    p {
+      color: $base-font-color;
+      line-height: $base-line-height;
+      margin: 0;
+    }
+
+    &:after {
+      @include position(absolute, null 0 null 0);
+      border: $tooltip-arrow-width solid transparent;
```

Comment:
Color literals like `transparent` should only be used in variable declarations; they should be referred to via variable everywhere else.